src/microbe_stage/MicrobeHUD.cs

```diff
@@ -1076,9 +1076,6 @@ private void SetEditorButtonFlashEffect(bool enabled)
     private void UpdatePopulation()
     {
         populationLabel.Text = stage!.GameWorld.PlayerSpecies.Population.FormatNumber();
-
-        // Reset box height
-        populationLabel.GetParent<Control>().MarginTop = 0;
```

Comment:
This looks a bit suspicious, but I'll assume we don't need this anymore.

Reply:
Yep, I've found a better solution using the right arrangement of controls.

Reply:
Sounds good.